core/chains/evm/logpoller/orm_test.go

```diff
@@ -68,6 +67,7 @@ func GenLogWithData(chainID *big.Int, address common.Address, eventSig common.Ha
 
 func TestLogPoller_Batching(t *testing.T) {
 	t.Parallel()
+	ctx := context.Background()
```

Comment:
Just marking another spot
```suggestion
	ctx := testutils.Context(t)
```